src/templates/readme.md

```diff
@@ -0,0 +1,42 @@
+# API Mesh Starter Kit
```

Comment:
This is the sample readme from the quick start guide. We need to change this for the init command output. @jhadobe I could use your help to modify this.

Reply:
@revanth0212 please review this PR https://github.com/AdobeDocs/graphql-mesh-gateway/pull/216. The readme in this repo should link to the documentation site after it is published.

Reply:
Do you have a potential link for the published doc page? Ill add it the readme and the link will be active once the docs are published. Anyways these 2 PRs will go live at the same time.

Also, what do you think the contexts of this readme should be? Just a heading and a link to the doc page or something else?

Reply:
@revanth0212 it depends when you want this page to publish. i'm waiting on some url forwarding. the destination will be: https://developer.adobe.com/graphql-mesh-gateway/mesh/template (after the merge), which should take place in the next day or so (just waiting on the release to be published).

I think it should contain a brief overview of the files and their purpose, but yes, mainly a link to the docs.